### PR TITLE
[WIP]Test to automate task sorting

### PR DESCRIPTION
**Pull request by nachandr**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a4cc444fb9f684cfb9706d87963613f16fddba18:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/configure/tasks.py [hunk 1]
```diff
@@ -160,3 +160,7 @@ def all_other_tasks(**filter_kwargs):
     Returns: List of dicts.
     """
     return _get_tasks("tasks_all_other", **filter_kwargs)
+
+
+def sort_by_header(header, order):
```
Comment (psav) [inline cfme/configure/tasks.py]: I would prefer not to introduce this fn, and instead for the calls below to be

```from cfme.configure.tasks import tasks_table

tasks.sory_by('Started', 'ascending')
```

Comment (nachandr) [inline cfme/configure/tasks.py]: @psav , @mfalesni, In cfme/configure/tasks.py, tasks_table is an instance of CheckboxTable.
The CheckboxTable class has no sorting methods.The cfme/web_ui/__init__.py file
also has this: class SortTable(Table). I'd like to create a new class, SortCheckboxTable and then use
tasks_table as an instance of SortCheckboxTable.

I'd appreciate your thoughts on this.

**Comment by dajoRH** [comment]
Lint report for commit 4a1ea7dfab2525445d624fa3eeb69cf7e7f3c496:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/configure/tasks.py (hunk 1 above)
Comment (psav) [inline cfme/configure/tasks.py]: @mfalesni this is what #2771 was to do with?

**Comment by mshriver** [comment]
@nachandr Please update this branch and comment on the current state of the PR. I'm available for first-pass review if it is ready and still relevant.

**Comment by nachandr** [comment]
@mshriver I was waiting on PR#2771, but I believe I should be using the new Widgetastic table.

I plan on updating this PR next week

**Comment by mfalesni** [comment]
@mshriver The game plan for this should be roughly:

1) #4161 gets merged
2) The view that @nachandr uses gets converted to Widgetastic (or is it already?)
3) @nachandr updates this PR so it uses navmazing/widgetastic

**Comment by mfalesni** [comment]
@nachandr Hey, how is this one looking? Is it stale?

**Comment by jkrocil** [comment]
I believe this is potentially blocked by me; leaving open for now, will revisit.

**Comment by mshriver** [comment]
Please re-open PR if necessary

**Pull request closed without merging**